src/core_modules/capture-core/components/Widget/WidgetNonCollapsible.component.js

```diff
@@ -0,0 +1,43 @@
+// @flow
+import React, { type ComponentType } from 'react';
+import { withStyles } from '@material-ui/core';
+import { colors, spacersNum } from '@dhis2/ui';
+import type { WidgetNonCollapsibleProps, WidgetNonCollapsiblePropsPlain } from './widgetNonCollapsible.types';
+
+const styles = {
+    container: {
+        borderRadius: 3,
+        borderStyle: 'solid',
+        borderColor: colors.grey400,
+        borderWidth: 1,
+    },
+    header: {
+        display: 'flex',
+        alignItems: 'center',
+        padding: spacersNum.dp16,
+        fontWeight: 500,
+        fontSize: 16,
+        color: colors.grey800,
+    },
+};
+
+const WidgetNonCollapsiblePlain = ({ header, children, color, classes }: WidgetNonCollapsiblePropsPlain) => (
+    <div className={classes.container}>
```

Comment:
Any reason why we don't set `style={{ backgroundColor: color || colors.white }}` here (instead of setting it twice)?